Integrations/integration-Sandblast.yml

```diff
@@ -0,0 +1,249 @@
+description: Query, upload and download data using Check Point Sandblast
+configuration:
+- display: Server URL (e.g. https://192.168.0.1)
+  name: server
+  defaultvalue: https://te.checkpoint.com
+  type: 0
+  required: true
+- display: Version
+  name: version
+  defaultvalue: v1
+  type: 0
+  required: true
+- display: API key
+  name: token
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Do not validate server certificate (insecure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+script:
+  script: |-
+    var server = params.server;
+    var insecure = params.insecure;
+    var token = params.token;
+    var version = params.version;
+    var requestTemplate = JSON.stringify({request:[{features:['te']}]});
+
+    var base = server +'/tecloud/api/' + version + '/file/';
```

Comment:
I think you need to sanitize the server URL.
Check if it ends with a '/' or not ,,,